### PR TITLE
new

### PR DESCRIPTION
**Pull request by Saliha192**

### CODE REVIEW

File: src/main/java/org/apache/commons/collections4/IterableUtils.java [hunk 1]
```diff
@@ -203,11 +203,11 @@ public static <E> Iterable<E> chainedIterable(final Iterable<? extends E> a,
...
-    public static <E> Iterable<E> chainedIterable(final Iterable<? extends E> a,
-                                                  final Iterable<? extends E> b,
-                                                  final Iterable<? extends E> c,
-                                                  final Iterable<? extends E> d) {
-        return chainedIterable(new Iterable[] {a, b, c, d});
+    public static <E> Iterable<E> chainedIterable(final Iterable<? extends E> e,
+                                                  final Iterable<? extends E> f,
+                                                  final Iterable<? extends E> g,
+                                                  final Iterable<? extends E> h ) {
+        return chainedIterable();
```
Comment (aherbert) [inline src/main/java/org/apache/commons/collections4/IterableUtils.java:210]: It is unclear what this PR is for, but this is a bug as the args are now ignored.

File: pom.xml [hunk 2]
```diff
@@ -515,6 +522,13 @@
       <version>33.0.0-jre</version>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
```
Comment (garydgregory) [inline pom.xml:526]: Not needed, this comes in from the parent POM.

File: pom.xml [hunk 3]
```diff
@@ -491,6 +491,13 @@
       <version>5.2.0</version>
       <scope>test</scope>
     </dependency>
+
+      <dependency>
```
Comment (garydgregory) [inline pom.xml:495]: FIx indentation.

File: pom.xml [hunk 4]
```diff
@@ -673,6 +709,59 @@
           </archive>
         </configuration>
       </plugin>
+
+      <plugin>
+        <groupId>org.jacoco</groupId>
+        <artifactId>jacoco-maven-plugin</artifactId>
+        <version>0.8.11</version> <!-- Use the latest version available -->
```
Comment (garydgregory) [inline pom.xml:716]: jacoco-maven-plugin is defined in the parent POM. If you want to update the version, there is a property for that.

File: pom.xml [hunk 5]
```diff
@@ -741,16 +830,14 @@
...
       </plugin>
-      <plugin>
-        <groupId>com.github.spotbugs</groupId>
-        <artifactId>spotbugs-maven-plugin</artifactId>
-      </plugin>
+
      <plugin>
         <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-pmd-plugin</artifactId>
+       <artifactId>maven-pmd-plugin</artifactId>
```
Comment (garydgregory) [inline pom.xml:840]: Fix indentation.

File: pom.xml [hunk 6]
```diff
@@ -645,6 +659,27 @@
...
+
+      <plugin>
+        <groupId>com.github.spotbugs</groupId>
+        <artifactId>spotbugs-maven-plugin</artifactId>
+
+        <configuration>
+          <effort>Max</effort>
+          <threshold>medium</threshold>
+          <failOnError>true</failOnError>
+          <includeFilterFile>${basedir}/src/conf/spotbugs-include-filter.xml</includeFilterFile>
```
Comment (garydgregory) [inline pom.xml:671]: We already have `src/conf/findbugs-exclude-filter.xml`.

File: src/conf/spotbugs-exclude-filter.xml [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+
+<!--
+  This file contains some false positive bugs detected by findbugs. Their
+  false positive nature has been analyzed individually, and they have been
+  put here to instruct findbugs it must ignore them.
+-->
```
Comment (garydgregory) [inline src/conf/spotbugs-exclude-filter.xml:23]: We already have `src/conf/findbugs-exclude-filter.xml`.

**Review by garydgregory — CHANGES_REQUESTED** [review]
-1: This PR is confusing. The PR does not even have a description and introduces at least one bug; also the PR title is nonsensical.

**Comment by garydgregory** [comment]
Closing:  No reply.

**Pull request closed without merging**